src/main/java/com/github/shyiko/mysql/binlog/event/deserialization/EventDeserializer.java

```diff
@@ -307,6 +307,10 @@ public EventDataDeserializer getEventDataDeserializer(EventType eventType) {
         return eventDataDeserializer != null ? eventDataDeserializer : defaultEventDataDeserializer;
     }
 
+    public Map<Long, TableMapEventData> getTableMapEventByTableId() {
+        return tableMapEventByTableId;
```

Comment:
This allows user to get the `tableMapEventByTableId` back from the default EventDeserializer. Whenever a library user wants to selectively override any of the eventDeserializers, the library user would probably need to access this map.